PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/CustomerSurveyHelper.java

```diff
@@ -100,7 +101,11 @@ public void neverShowAgain() {
         sendTelemetry(TELEMETRY_VALUE_NEVER_SHOW);
     }
 
-    private boolean isAbleToPopUpSurvey() {
+    private synchronized boolean isAbleToPopUpSurvey() {
+        if (isShown) {
+            return false;
+        }
+        isShown = true;
```

Comment:
when to set isShown  to false? never?

Reply:
``` java
private boolean isShown = false;
```
And it will set to false once the window is pop up and will never show again unless users restart the IDE